packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -103,19 +111,19 @@ class SelectedFilters extends Component {
 	};
 
 	// determines whether any filter has been applied regardless of `showFilter=false`
-	hasFilters = () =>
-		Object.keys(this.props.selectedValues)
+	hasFilters = () => {
+		const componentProps = this.getStoreAccess().props;
```

Comment:
@mohdashraf010897 It seems wrong. You would get the redux store with `getState` method.

Reply:
`getStoreAccess = () => (this.context && this.context.storeState ? this.context.storeState : {});`

Have used context and defined a method to fetch it... Line 38

Reply:
ok